frontend/packages/noobaa-storage-plugin/src/components/object-data-reduction-card/object-data-reduction-card.tsx

```diff
@@ -55,8 +55,8 @@ const DataReductionCard: React.FC<DashboardItemProps> = ({
     'loadError',
   ]);
 
-  const efficiency = getGaugeValue(efficiencyQueryResult);
-  const savings = getGaugeValue(savingsQueryResult);
+  const efficiency = Number(getGaugeValue(efficiencyQueryResult));
+  const savings = Number(getGaugeValue(savingsQueryResult));
```

Comment:
not good approach to convert into the number and then check for NaN.
We should guard on the coming values instead of converting them to something.
```suggestion
  const savings = getGaugeValue(savingsQueryResult);
```

Reply:
If you think about it undefined gets converted to NaN anything else stays intact. So guarding against NaN is the same thing.